### PR TITLE
113 feature/capture email result

### PR DESCRIPTION
**Pull request by mariadanieldeepak**
Closes https://github.com/sudar/email-log/issues/113

### CODE REVIEW

File: include/Core/DB/TableManager.php [hunk 1]
```diff
@@ -270,6 +271,87 @@ public function get_logs_count() {
...
+		if ( empty( $data ) || ! is_array( $data ) ) {
+			$query_cond .= ' WHERE id = 0';
+		}
+
+		if ( array_key_exists( 'to', $data ) ) {
+			$to_email   = trim( esc_sql( $data['to'] ) );
+			$query_cond .= " WHERE to_email = '$to_email'";
+		}
+
+		if ( array_key_exists( 'subject', $data ) ) {
```
Comment (sudar) [inline include/Core/DB/TableManager.php]: This could lead to a syntax error if `to` is not present in the data array.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,1468 @@
+{
```
Comment (sudar) [inline package-lock.json]: No need to commit this file.

File: include/Core/DB/TableManager.php (hunk 1 above)
Comment (sudar) [reply include/Core/DB/TableManager.php]: I am still seeing this issue. Can you explain how it is fixed?

**Comment by mariadanieldeepak** [comment]
@sudar,

I tried couple of scenarios and found the query to give expected results.

image

image

image

image

**Comment by mariadanieldeepak** [comment]
@sudar,

I've fixed the following as discussed in the call.

* Used [Early return](https://github.com/sudar/email-log/pull/163/commits/b1bec7b084fb5cce76d300396e45c6bdde3bb3a8) pattern
* Indented the code for improved readability (Refer [this](https://github.com/sudar/email-log/pull/163/commits/4dc0d55713d2eeb6031c933d44f1748f3046b498) commit)

**Comment by sudar** [comment]
@mariadanieldeepak 

Code looks good and I have approved it.

But there are some issues with Style CI. Once it is fixed, we can merge this.

**Comment by mariadanieldeepak** [comment]
@sudar,

> But there are some issues with Style CI. Once it is fixed, we can merge this.

Fixed the Style CI issues.

**Pull request merged**